Sources/SwifterSwift/SwiftStdlib/BinaryIntegerExtensions.swift

```diff
@@ -0,0 +1,49 @@
+// BinaryIntegerExtensions.swift - Copyright 2021 SwifterSwift
+
+#if canImport(Foundation)
```

Comment:
This has no foundation dependency as far as I can see, so maybe we don't need to import Foundation nor the conditional check? :)